ddht/v5_1/app.py

```diff
@@ -106,11 +106,12 @@ async def run(self) -> None:
         self.logger.info("Protocol-Version: %s", self._boot_info.protocol_version.value)
         self.logger.info("DDHT base dir: %s", self._boot_info.base_dir)
         self.logger.info("Starting discovery service...")
-        self.logger.info("Listening on %s:%d", listen_on, port)
```

Comment:
why was the port dropped? that seems like it could be useful!

Reply:
Not dropped.  It was being double reported.  `listen_on` is an `Endpoint` which contains both IP and port in a single object.